action.yaml

```diff
@@ -21,6 +21,10 @@ inputs:
   redmine_api_key:
     description: The Redmine API key
     required: true
+  slack_webhook_url:
```

Comment:
Not strictly required by the ticket but maybe you can just make it `webhook_url` since all it does is post. There's nothing Slack-specific here besides the `msg`. And that could even be an option to make it useful for other cases.